core/services/vrf/v2/integration_v2_test.go

```diff
@@ -2105,6 +2105,7 @@ func TestStartingCountsV1(t *testing.T) {
 			RequestID: &reqID3,
 		})
 		require.NoError(t, err2)
+		md_ := sqlutil.JSON(md)
```

Comment:
I see that trailing underscores is already the local style here, but that seems odd. What about naming them something like:
```suggestion
		mdSQL := sqlutil.JSON(md)
```

Reply:
I agree that looks better. I've updated it in the latest commit.